saas/urls/provider/metrics.py

```diff
@@ -42,6 +42,8 @@
         name='saas_subscriber_pipeline_download'),
     url(r'^plans/((?P<from_date>\d\d\d\d-\d\d)/)?',
         PlansMetricsView.as_view(), name='saas_metrics_plans'),
+    url(r'^coupons/_download/?$', CouponMetricsDownloadView.as_view(),
```

Comment:
Is it a convention to use an '_' prefix for download links? Should we do that in other parts of the code base?